pkg/controllers/csi/livenessprobe/livenessprobe.go

```diff
@@ -0,0 +1,138 @@
+package livenessprobe
```

Comment:
TBH i don't see any benefits to put it to `pkg/controllers` it's nothing related to controllers IMO.

Reply:
right, moved to `pkg/csi`